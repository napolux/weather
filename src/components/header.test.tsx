import React from 'react';
import WeatherHeader from './header';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
    const tree = renderer
        .create(
            <WeatherHeader />
        )
        .toJSON();
    expect(tree).toMatchSnapshot();
});