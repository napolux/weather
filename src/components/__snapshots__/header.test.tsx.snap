// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="ui container"
>
  <div
    className="ui menu"
  >
    <div
      className="header item"
      onClick={[Function]}
    >
      Weather
    </div>
    <div
      className="item"
      onClick={[Function]}
    >
      <p>
        A project by 
        <a
          href="https://napolux.com"
        >
          Francesco Napoletano
        </a>
      </p>
    </div>
  </div>
</div>
`;
